generate.py

```diff
@@ -0,0 +1,116 @@
+from pathlib import Path
+
+from aind_data_schema_models.generators._base import (
+    _HarpDeviceTypeModel,
+    _ModalityModel,
+    _PlatformModel,
+    _SpeciesModel,
+    _MouseAnatomyModel,
+    _RegistryModel,
+)
+from aind_data_schema_models.generators.generators import (
+    GeneratorContext,
+    MappableReferenceField,
+    ModelGenerator,
+    ForwardClassReference,
+)
+from aind_data_schema_models.generators.parsers import csv_parser, get_who_am_i_list
+
+if __name__ == "__main__":
+
+    root = Path(__file__).parent / r"src/aind_data_schema_models/models"
```

Comment:
I think this might not translate across different operating systems? Maybe
```python
root = Path(__file__).parent / "src" / "aind_data_schema_models" / "models"
```

Reply:
It should. The Path class should gracefully handle different operating systems. Keep in mind that the "/" is an overload of the `__truediv__` dunder operator not a literal.